test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	/*
+	 * This method inputs a null object and tests IsAnyNull and should return true
+	 */
+	@Test
+	public void testIsAnyNull_NullObject() {
```

Comment:
The method name should not include test. Should be something like isAnyNull_emptyList_falseReturned